source/sass/mixins/_typography.scss

```diff
@@ -120,5 +120,24 @@
 @mixin list-style-none() {
   @include padding(0, inline-start);
   // Overflow auto is so the bullet will be still read from a screen reader and will push the bullet off screen
-  overflow: auto;
+  overflow-x: auto;
+}
+
+@mixin comma-separator() {
```

Comment:
Would prefer `list-separator` or something (with an option for `alternative`?).

Wonder if can also have a mixin for making inline lists generally?

Reply:
👍 for `list-separator`.

Mixin for list generation seems overkill at this stage. Happy to revisit if we find a few more examples of variation in the future.